webpack.config.js

```diff
@@ -22,25 +22,19 @@ entries.unshift('core-js/fn/symbol');
 entries.unshift('whatwg-fetch');
 entries.unshift('url-search-params-polyfill');
 
-const basePath = 'build/admin';
-
 module.exports = (env, argv) => ({ // eslint-disable-line no-undef
     entry: entries,
     output: {
-        path: path.resolve('public'),
-        filename: basePath + '/[name].[chunkhash].js',
+        path: path.resolve('public/build/admin'),
+        filename: '[name].[chunkhash].js',
     },
     devtool: argv.mode === 'development' ? 'eval-source-map' : 'source-map',
     plugins: [
         new webpack.DefinePlugin({
             BUNDLE_ENTRIES_COUNT: entriesCount,
         }),
-        new MiniCssExtractPlugin({
-            filename: basePath + '/[name].[chunkhash].css',
-        }),
-        new ManifestPlugin({
-            fileName: basePath + '/manifest.json',
-        }),
+        new MiniCssExtractPlugin(),
```

Comment:
This way the `chunkhash` is missing, and therefore the cache busting would not work.